packages/neuron-ui/src/components/NervosDAO/DepositDialog.tsx

```diff
@@ -52,7 +65,7 @@ const DepositDialog = ({
     ),
     []
   )
-  const maxValue = +(BigInt(balance) / BigInt(SHANNON_CKB_RATIO)).toString()
+  const maxValue = +(maxDepositAmount / BigInt(SHANNON_CKB_RATIO)).toString()
```

Comment:
Why convert this value to number ?

Reply:
> Why convert this value to number ?

The component of Fabric UI only accepts number type, neither bigint nor string.

Reply:
OK, please make sure it has no precision problems.